.github/workflows/docker_publish.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
             type=semver,pattern={{ major }}.{{ minor }},enable=${{ github.event_name == 'workflow_dispatch' && inputs.version != '' }},value=${{ inputs.version }}
             type=semver,pattern={{ major }},enable=${{ github.event_name == 'workflow_dispatch' && inputs.version != '' }},value=${{ inputs.version }}
             # only enabled in main:
-            type=edge,prefix=main-,suffix=-${{ matrix.version }},enable=${{ github.ref == 'refs/heads/main' }}
+            type=edge,prefix=main-,enable=${{ github.ref == 'refs/heads/main' }}
```

Comment:
For reference on type edge: https://github.com/docker/metadata-action/tree/v5/?tab=readme-ov-file#typeedge